examples/braket_features/Using_the_experimental_local_simulator.ipynb

```diff
@@ -0,0 +1,362 @@
+{
```

Comment:
<p>Technically, the older simulator also supports up to 32 qubits; the limitation is due to numpy allowing at most 32-dimensional arrays.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>